packages/internal/src/build/api.ts

```diff
@@ -50,8 +50,16 @@ export const prebuildApiFiles = (srcFiles: string[]) => {
       return undefined
     }
 
+    // Fixes nodeFileTrace errors on windows when building
+    let nodeFileTraceCompliantCode: string = result.code
+    if (process.platform === 'win32') {
+      nodeFileTraceCompliantCode = nodeFileTraceCompliantCode.replaceAll(
+        /\\\\/g,
```

Comment:
```suggestion
        '\\\\',
```

You're already doing `replaceAll`, so a regexp with the `g` modifier isn't needed, right?

Reply:
You are right. That was from me trying to fix it further upstream.
I retested on my machine and regex was not needed.